src/coreclr/nativeaot/Runtime/unix/unixasmmacrosamd64.inc

```diff
@@ -320,7 +320,7 @@ C_FUNC(\Name):
 DEFAULT_FRAME_SAVE_FLAGS = PTFF_SAVE_ALL_PRESERVED + PTFF_SAVE_RSP
 
 .macro PUSH_COOP_PINVOKE_FRAME trashReg
-    push_nonvol_reg rbp                         // push RBP frame  // TODO: do we need this? not on windows.
```

Comment:
I have checked and Windows debugger does not have issues with unwinding without introducing rbp frame, so I did not change anything.